models/enterprise_sonic/snmp/deleted_example_01.txt

```diff
@@ -1,28 +1,30 @@
 # Using deleted
-
+#
 # Before state:
-# -------------
+#---------------
 #
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# show running-configuration | grep snmp
 #
-#         Community Name                     Group Name
-#-------------------------------- --------------------------------
-#private                          ro
-#public                           ro
+# snmp-server user user1 group group-lab encrypted auth
+# md5 auth-password
+# U2FsdGVkX18J+L+L9OyQYWpAkGUrTgcg/6xzZoDjCbQw1ISHJ5mxmxrYZgQypEUXDeNe6rBupsc9sVDJBKxrwA==
+# priv aes-128 priv-password U2FsdGVkX1/Xs+ffZvdV9YzfyGHgIJ+zkLRPfF3/WgYIE1S4Ribvbzhu5chpHHI7ooCBpcVxYZotAXDzgetxvQ==
 #
-- name: Delete a single SNMP community
-  sonic_snmp:
-    config:
-      - index: "private"
-        securityName: "ro"
-    state: delete
 
-# After state:
-# ------------
-#
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# - name: Delete snmp-server configuration
```

Comment:
There should be at least two "deleted" examples:

- one to illustrate deletion of specific options.
- one to illustrate deletion of all snmp server configuration.

For this one, highlight the fact that it is for specific options:

```suggestion
# - name: Delete specific options from snmp-server configuration
```

Reply:
You can mark this Issue "Resolved". (Although there are some new issues posted for the "deleted" examples, the current structure is a step toward defining the two types of deletion, and the new comments can be used to track the remaining changes.)